app/labor/random_gif.rb

```diff
@@ -19,8 +19,7 @@ def initialize(_action = "congratulations")
       "l0K4glBiv82lZ0Zuo" => { aspect_ratio: 0.563 },
       "7EcgJbeY0yCRy" => { aspect_ratio: 0.750 },
       "Gf3fU0qPtI6uk" => { aspect_ratio: 0.750 },
-      "5GoVLqeAOo6PK" => { aspect_ratio: 0.780 },
-      "82lalqBsmW56Z6E2oV" => { aspect_ratio: 0.562 }
```

Comment:
I tossed this in because that giphy ID was broken. Ideally it's its own PR but I through it in because I thought it was a non-controversial add-on. Should have made note.